server/sonar-web/src/main/js/apps/component-measures/drilldown/MeasureCell.js

```diff
@@ -21,23 +21,26 @@
 import React from 'react';
 import Measure from '../../../components/measure/Measure';
 import { isDiffMetric } from '../../../helpers/measures';
-/*:: import type { Component } from '../types'; */
+/*:: import type { ComponentEnhanced } from '../types'; */
+/*:: import type { MeasureEnhanced } from '../../../components/measure/types'; */
 /*:: import type { Metric } from '../../../store/metrics/actions'; */
 
 /*:: type Props = {
-  component: Component,
+  component: ComponentEnhanced,
+  measure?: MeasureEnhanced,
   metric: Metric
 }; */
 
-export default function MeasureCell({ component, metric } /*: Props */) {
+export default function MeasureCell({ component, measure, metric } /*: Props */) {
+  const getValue = (item /*: { leak?: ?string; value?: string } */) =>
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove parentheses around the parameter of this arrow function. ([javascript:S3524](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fstas%2Ffix-measure&issues=AWFRCUmnsS4rYBZqFAm9&open=AWFRCUmnsS4rYBZqFAm9)

[ISSUE_KEY]: AWFRCUmnsS4rYBZqFAm9